Lib/Src/CAdcData.cpp

```diff
@@ -14,7 +14,8 @@
 #define ADC_START_BIT        2
 #define ADC_CONVERSION_COEFF ((float)ADC_VDDA / ADC_RES)
 
-CUartCom uart_adc(&huart2);
+// TODO: Delete this instance
+CUartCom uart_adc("ADC Data");
```

Comment:
Should instantiate the `CUartComs` as a global variable in `cpp_main.cpp`. This way they can be linked to `CDispatcher` class instance.